### DISCUSSION TITLE
Supabase studio without supabase

### DISCUSSION THREAD

**Discussion by ansarizafar** [supabase/supabase#4878] [Questions]
Is it possible to use studio with a standalone Postgres instance as an alternative to pgadmin?

**Comment by soedirgo** [comment]
You can of course point it to a standalone Postgres db, but Studio expects some Supabase-specific schema objects, so some things won't work properly (auth, auto-generated API).

Things like SQL editor and table editor should work fine, but it's untested so expect some hiccups.
